R/serialized_tests.R

```diff
@@ -160,23 +165,21 @@ style_empty <- function(text) {
 
 #' @describeIn test_transformer Transformations for indention based on operators
 style_op <- function(text) {
-
   transformers <- list(
     # transformer functions
     initialize = default_style_guide_attributes,
-    line_break = NULL,
-    space      = partial(indent_op, indent_by = 2),
-    token      = NULL,
+    line_break        = NULL,
```

Comment:
Are these manual changes? Need to align with `initialize` perhaps.

Reply:
Yes. Agree.